eth/downloader/downloader_test.go

```diff
@@ -1490,7 +1490,6 @@ func testFakedSyncProgress(t *testing.T, protocol int, mode SyncMode) {
 // This test reproduces an issue where unexpected deliveries would
 // block indefinitely if they arrived at the right time.
 func TestDeliverHeadersHang(t *testing.T) {
-	t.Parallel()
```

Comment:
You removed this -- was that intentional?

Reply:
Yes, because we call t.Parallel() in the subtests already, calling it twice doesn't really make sense

Reply:
Ah, but those are different. 
This one means
 - This test can be run in parallel (with other tests in this module) 
The other one means
- These subtests can be run in parallel (with eachother)

If you call `Parallel` twice on a `t`, it'll panic btw, so there's never any calling it twice